draft-ietf-quic-transport.md

```diff
@@ -786,15 +775,30 @@ reaches 2^62 - 1, the sender MUST close the connection without sending a
 CONNECTION_CLOSE frame or any further packets; a server MAY send a Stateless
 Reset ({{stateless-reset}}) in response to further packets that it receives.
 
-For the packet header, the number of bits required to represent the packet
-number are reduced by including only the least significant bits of the packet
-number.
-
-The encoded packet number is protected as described in {{QUIC-TLS}}. Protection
-of the packet number is removed prior to recovering the full packet number. The
-full packet number is reconstructed at the receiver based on the largest packet
-number received on a successfully authenticated packet. Recovering the full
-packet number is necessary to successfully remove packet protection.
+In the QUIC long and short packet headers, the number of bits required to
+represent the packet number are reduced by including only the least
+significant bits of the packet number.
+
+In the long packet header, the least significant 32 bits are used. In the
+short packet header, the number of significant bits are encoded in the most
+significant two bits of the first octet of the encoded packet number:
+
+| First octet pattern | Encoded Length | Bits Present |
+|:--------------------|:---------------|:-------------|
+| 0b0xxxxxxx          | 1 octet        | 7            |
+| 0b10xxxxxx          | 2              | 14           |
+| 0b11xxxxxx          | 4              | 30           |
+
+Note that these encodings are similar to those in {{integer-encoding}}, but
+use different values.
+
+The encoded packet number is protected as described in Section 5.6 {{QUIC-TLS}}.
+Protection of the packet number is removed prior to recovering the full packet
+number.The full packet number is reconstructed at the receiver based on the
+value of the most significant two bits in the first octet (to determine the
```

Comment:
This sentence is a bit confusing, I think the PN is decoded based on the number of significant bits and the number those significant bits encode.  Saying it's based on the value of the first two bits more of a wire format detail I don't think needs to be in this sentence.

Reply:
I like Ian's suggestion too.